### PR TITLE
Add documents for dyson climate device

### PR DESCRIPTION
**Pull request by soraxas**
**Description:**
Documentation for the now supported dyson climate platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14598

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Looks good to me @soraxas! Thanks a lot!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Merge conflicts have appeared!
Could you please take a look @soraxas

**Comment by frenck** [comment]
Thanks for resolving the conflict @soraxas! 👍

**Review by frenck — DISMISSED** [review]

File: source/_components/climate.dyson.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+footer: true
+logo: dyson.png
+ha_category: Climate
+ha_iot_class: "Cloud Polling"
+ha_release: 0.77
+---
+
+The `dyson` climate platform allows you to control your Dyson Pure Hot+Cool Fan thermal control. For controlling the fan functionality, see the [dyson fan](/components/fan.dyson/) component.
+
+You have first to set up the [Dyson component](/components/dyson/)
```
Comment (tjorim) [inline source/_components/climate.dyson.markdown]: Please put a point at the end of a sentence.

File: source/_components/climate.dyson.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### {% linkable_title Component services %}
+
+This component support the following services (see [Climate](/components/climate/)):
```
Comment (tjorim) [inline source/_components/climate.dyson.markdown]: This component support --> support _**s**_

**Review by tjorim — CHANGES_REQUESTED** [review]
Only some typo's.

⚠️ The ha_release will need an update before merging this one in.

**Review by frenck — CHANGES_REQUESTED** [review]
Merge conflict again 😢

**Comment by soraxas** [comment]
Thanks @frenck :)

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/climate.dyson.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+description: "Instructions on how to integrate your Dyson Climate device within Home Assistant."
+date: 2018-05-23 22:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dyson.png
+ha_category: Climate
+ha_iot_class: "Cloud Polling"
+ha_release: 0.77
```
Comment (MartinHjelmare) [inline source/_components/climate.dyson.markdown]: 0.81

File: source/_components/climate.dyson.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+The `dyson` climate platform allows you to control your Dyson Pure Hot+Cool Fan thermal control. For controlling the fan functionality, see the [dyson fan](/components/fan.dyson/) component.
```
Comment (MartinHjelmare) [inline source/_components/climate.dyson.markdown]: ... see the dyson fan platform.

**Review by frenck — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**